src/db.rs

```diff
@@ -1093,6 +1093,15 @@ impl DB {
         DBIterator::new(self, &readopts, mode)
     }
 
+    pub fn iterator_cf_opt(
```

Comment:
@elichai It would be nice to add some description for public method

Reply:
Done.